### ISSUE TITLE
API Gateway usage plan resource does not support tags

### ISSUE THREAD

**Issue by zihaoyu** [terraform-providers/terraform-provider-aws#11146]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

0.12.6

### Affected Resource(s)

* aws_api_gateway_usage_plan

### Terraform Configuration Files

```hcl
resource "aws_api_gateway_usage_plan" "throttle" {
  name         = "${var.usage_plan_name}"

  api_stages {
    api_id = "${var.api_id}"
    stage = "${var.stage}"
  }

  throttle_settings {
    burst_limit = "${var.burst_limit}"
    rate_limit  = "${var.rate_limit}"
  }

  tags = {
    environment = "${var.environment}"
    region      = "${var.region}"
  }
}
```

### Debug Output

Output from `terratest`:

```
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158: Error: Unsupported argument
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158:
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158:   on main.tf line 19, in resource "aws_api_gateway_usage_plan" "throttle":
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158:   19:   tags = {
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158:
TestCreateUsagePlan 2019-12-04T16:55:06-05:00 command.go:158: An argument named "tags" is not expected here.
```

### Expected Behavior

Usage plan is created with provided tags.

### Actual Behavior

Terraform plan fails.

### Steps to Reproduce

1. `terraform apply`

### References

* #10566 

Looks like `tags` was not added to `aws_api_gateway_usage_plan`'s [`Schema`](https://github.com/terraform-providers/terraform-provider-aws/blob/v2.41.0/aws/resource_aws_api_gateway_usage_plan.go#L26).

**Comment by bflad** [comment]
Hi @zihaoyu 👋 Thank you for reporting this and sorry you ran into trouble here. This appears to have been caused by some unexpected behavior with the Squash and Merge functionality within GitHub. We'll get this addressed and released in the next version (2.42.0) of the Terraform AWS Provider.

**Comment by zihaoyu** [comment]
Thank you @bflad ! Looking forward to the release.
